src/NUnitFramework/framework/Internal/OSPlatform.cs

```diff
@@ -291,7 +291,17 @@ public bool IsWinCE
             get { return platform == PlatformID.WinCE; }
         }
 
-#if !NETCF
+#if NETCF
+        /// <summary>
+        /// Return true if the platform is Xbox
+        /// </summary>
+        public bool IsXbox { get { return false; } }
```

Comment:
Since we add UnixPlatformID_Microsoft and UnixPlatformID_Mono as out own defines to handle builds < 3.5, why not add MaxOSXPlatformID as a static value in OSPlatform as well.  It did not exist before 3.5 either.  Then, no conditional would be required.